apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/LoggerOptions.cs

```diff
@@ -103,14 +119,87 @@ public sealed class LoggerOptions
             MonitoredResource monitoredResource = null,
             BufferOptions bufferOptions = null,
             RetryOptions retryOptions = null,
-            TextWriter loggerDiagnosticsOutput = null)
+            TextWriter loggerDiagnosticsOutput = null) =>
+            CreateWithServiceContext(logLevel, logName, labels, monitoredResource, bufferOptions, retryOptions, loggerDiagnosticsOutput);
```

Comment:
Why do we not want to use CreateWithDefaultServiceContext here?

Reply:
Yup, that works for me.

Reply:
Except that "default" doesn't actually work as a name, given that it's not what happens by default...